web-basics-browser-security-model/activities/cockroach/deploy/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.8
```

Comment:
It would be better, but it would be necessary to rewrite all Dockerfile since this image makes all the work behind.
I would see this as an improvement, rather than the scope of this PR, but your call, let me know if you want to include this change in this PR, otherwise, mark this conversation as solved and create an issue with this.